templates/pages/seller-account.hypr

```diff
@@ -0,0 +1,42 @@
+﻿{% extends "page" %}
+
+{% block title-tag-content %}{% firstof pageContext.metaTitle labels.yourAccount %} - {% parent %}{% endblock title-tag-content %}
+
+{% block body-tag-classes %} mz-myaccount {% endblock body-tag-classes %}
+
+{% block body-content %}
+
+{% require_script "pages/selleraccount" %}
+
+{% preload_json model "customer" %}
+
+<div class="mz-l-container">
+    <h1 class="mz-pagetitle">
+        {{ labels.yourAccount }}
+    </h1>
+
+    <div class="mz-l-sidebar mz-myaccount-nav">
+        {% include "modules/my-account/seller-account-nav" with navtarget="body" %}
+    </div>
+
+    <div id="account-panels" class="mz-myaccount-panels mz-l-stack mz-l-stack-large">
```

Comment:
Can you clean up the blank lines, tabs/spaces, and make it look a little nicer? Replace tabs with spaces.